src/engine/loaders/obj_loader.cpp

```diff
@@ -71,31 +71,140 @@ void ObjLoader::load(MeshFrame *o_result, char *t_filename, float t_scale, u8 t_
+
+                int* x;
+                fpos_t start;
+                fgetpos(file,&start);
+                        int matches = fscanf(file, "%d/%d/%d %d/%d/%d %d/%d/%d\n",x,x,x,x,x,x,x,x,x);
+                fsetpos(file,&start);
+                int newerMatches =0;
+
+                switch(matches)
+                {
+                    /* Vs, VTs and VNs all set */
+                    case 9:
+                    {
+                        fscanf(file, "%d/%d/%d %d/%d/%d %d/%d/%d\n",
+                                            &vertexIndex[0], &coordIndex[0], &normalIndex[0],
+                                            &vertexIndex[1], &coordIndex[1], &normalIndex[1],
+                                            &vertexIndex[2], &coordIndex[2], &normalIndex[2]);
+
+                    }
+                    break;
+                    /* Loaded only two digits (V, VT) succesfuly. Not setting VN. */
+                    case 2:
+                    {
+                        fscanf(file, "%d/%d/ %d/%d/ %d/%d/\n",
+                                            &vertexIndex[0], &coordIndex[0],
+                                            &vertexIndex[1], &coordIndex[1],
+                                            &vertexIndex[2], &coordIndex[2]);
+                        
+                    }
+                    break;
+                    /* Only V set. Checking for existance of VT or VN. */
+                    case 1:
+                    {
+                        /* Check for existance of V/// configuration.. */
+                        newerMatches = fscanf(file, "%d/// %d/// %d///\n",x,x,x);
+                        fsetpos(file,&start);
+                        if(newerMatches==3){
+                            /* Configuration confirmed. */
+                            fscanf(file, "%d/// %d/// %d///\n",&vertexIndex[0],&vertexIndex[1],&vertexIndex[2]);
+
+                        }else
+                        {
+                            /* Failed, checking configuration V//VN */
+                            newerMatches = fscanf(file, "%d//%d %d//%d %d//%d",x,x,x,x,x,x);
+                            fsetpos(file,&start);
+                            if(newerMatches==6){
+                                /* Configuration confirmed. */
+                                newerMatches = fscanf(file, "%d//%d %d//%d %d//%d",
+                                        &vertexIndex[0], &normalIndex[0],
+                                        &vertexIndex[1], &normalIndex[1],
+                                        &vertexIndex[2], &normalIndex[2]);
+                            }
+                            else{
+                                /*Unknown configuration.*/
+                                PRINT_ERR("Unknown .obj face for .obj file!");
+                                printf("Obj file mentioned: \"%s\"\n",path);
```

Comment:
This `printf()` will never occur. Look at the implementation of `PRINT_ERR()`

Reply:
Done.